terraform/vars.tf

```diff
@@ -13,6 +13,13 @@ variable "region" {
   type = string
 }
 
+#=================================================
+#  EMAILS NOTIFICATIONS
+#=================================================
+variable "new_user_notification_email_address" {
+  type = string
+}
```

Comment:
All Terraform variables should have a `description` property.

Reply:
@jsf9k - @dylanj1752 added those description fields, can you take a look at this again?

Reply:
_All_ Terraform variables should have a `description` property, not just the one created in this PR.  That said, I'll go ahead and approve if someone creates an issue in this repo to add the `description` properties later; I just don't want it to be forgotten.

Reply:
@jsf9k I have created an issue https://github.com/cisagov/domain-manager-cicd/issues/9 and assigned it to myself.